api/tests/acceptance/application/certification-centers/index_test.js

```diff
@@ -94,6 +94,7 @@ describe('Acceptance | Route | Certification Centers', function () {
             attributes: {
               email: 'some.user@example.net',
               lang: 'fr-fr',
+              role: 'ADMIN',
```

Comment:
Ce serait plus simple pour la modification de la valeur de le faire à un seul endroit, càd le modèle `CertificationCenterInvitation`.

```suggestion
              role: CertificationCenterInvitation.Roles.ADMIN,
```